docs/sources/configure-client/grafana-agent/go_pull.md

```diff
@@ -1,258 +1,224 @@
 ---
 title: "Go (pull mode)"
 menuTitle: "Go (pull mode)"
-description:  "Instrumenting Golang applications for continuous profiling"
+description: "Set up Go profiling in pull mode"
 weight: 10
 ---
 
 # Go (pull mode)
```

Comment:
I really like the description you used as a page title and heading. What do you think about using it as the page title and heading? 
```suggestion
# Set up Go profiling in pull mode
```